core/src/apps/ethereum/definitions_constants.py

```diff
@@ -9,7 +9,7 @@
     b"\xb8\xd2\xb2\x1d\xe2\x71\x24\xf0\x51\x1f\x90\x3a\xe7\xe6\x0e\x07\x96\x18\x10\xa0\xb8\xf2\x8e\xa7\x55\xfa\x50\x36\x7a\x8a\x2b\x8b",
 )
 
-MIN_DATA_VERSION = 1720519386
+MIN_DATA_VERSION = 1724682490
```

Comment:
Acknowledging that this timestamp is a little bit older than the current latest (`1725965878`).